pkg/proc/variables_test.go

```diff
@@ -2053,3 +2053,59 @@ func TestCapturedVarVisibleOnFirstLine(t *testing.T) {
 		}
 	})
 }
+
+func TestIssue4116(t *testing.T) {
```

Comment:
We decided a while back that we are no longer going to make tests named after issues because it's hard to remember what they are testing so change this `TestEmbeddedStructMethodsAndFieldLookup` and add this comment on the first line:

```
// See issue #4116
```